test/setup_environment.sh

```diff
@@ -258,6 +263,86 @@ if [[ -n ${build_ledger} ]]; then
     cd ..
 fi
 
+if [[ -n ${build_jade} ]]; then
+    # Testing with Jade tag 0.1.30
+    JADE_RELEASE_TAG="0.1.30"
```

Comment:
In 67793345a98b95fdb045ee733aac3af0c34a3261 "Add tests for Jade hww using qemu emulator"

We generally prefer to test on the development branch at whatever the latest commit is. This allows us to be aware of any breaking changes before a release is made. It also means that this doesn't need to be updated every time a new firmware release is made.

Reply:
Bold!  Ok, I may have to make a change in the Jade repo so we can easily extract the relevant commits of the esp-idf toolkit and the qemu emulator that are relevant to that jade commit.  Leave it with me.